docs/core/stories.rst

```diff
@@ -51,7 +51,46 @@ This is what we call a **story**.
   and contain the name of the action.
 - Events returned by an action are on lines immediately after that
   action. For example, if an action returns a ``SetSlot`` event,
-  this is shown as the line ``- slot{"slot_name": "value"}``
+  this is shown as the line ``- slot{"slot_name": "value"}``.
```

Comment:
```suggestion
  this is shown as ``slot{"slot_name": "value"}``.
```
I know the dash is correct formatting, but since this is in the middle of a sentence it doesn't add to the explanation